doc/input-output-reference/src/overview/group-airflow.tex

```diff
@@ -1636,22 +1548,17 @@ \subsubsection{Inputs}\label{inputs-7-003}
 
 \paragraph{Field: Door Protection Type}\label{field-door-protection-type}
 
-This field is a key/choice field that tells how the door is protected. The impact of this choice is decribed in the Engineering Reference. The key/choices are:
+This field is a key/choice field that tells how the door is protected. The impact of this choice is described in the Engineering Reference. The key/choice options are:
 
-\begin{itemize}
-\tightlist
-\item
-  None
-\end{itemize}
+\begin{description}
 
-None, or no door protection is the default choice
+\item[None] None, or no door protection is the default choice;
 
-\begin{itemize}
-\item
-  AirCurtain
-\item
-  StripCurtain
-\end{itemize}
+\item[AirCurtain] Air Curtain;
+
+\item[StripCurtain] Strip Curtain.
+
+\end{description}
```

Comment:
And right below here also. We must be doing a lot of cut-n-paste when writing these sections.

![image](https://user-images.githubusercontent.com/5778156/189993059-18fa2c21-4b10-400c-9ce0-1aa4ce915482.png)

Reply:
Thanks! Now these new batches are now corrected.